lib/katello/tasks/reindex.rake

```diff
@@ -36,6 +36,9 @@ namespace :katello do
       begin
         yield
       rescue Exception => e
+        if object_class == Katello::Erratum || Katello::Util::Search.pulp_backend_search_classes.include?(object_class)
```

Comment:
Will this have to be updated everytime we convert an object from ES to the database?